android/src/main/java/capacitor/plugin/appsflyer/sdk/AppsFlyerPlugin.kt

```diff
@@ -517,6 +535,37 @@ class AppsFlyerPlugin : Plugin() {
 
     }
 
+    @PluginMethod(returnType = PluginMethod.RETURN_NONE)
+    fun enableTCFDataCollection(call: PluginCall) {
+        val shouldEnable = call.getBoolean(AF_ENABLE_TCF_DATA_COLLECTION)
+        if (shouldEnable != null) {
+            AppsFlyerLib.getInstance().enableTCFDataCollection(shouldEnable)
+        } else {
+            call.reject("Missing boolean value $AF_ENABLE_TCF_DATA_COLLECTION")
+        }
+    }
+
+    @PluginMethod(returnType = PluginMethod.RETURN_NONE)
+    fun setConsentData(call: PluginCall) {
+        val consentData = call.getObject("data") ?: return with (call) {
```

Comment:
You don't need the `with` block, you can simply use `return call.reject("Missing consent data")`